client/test/bpmn-io-modelers/dmn/DmnModelerSpec.js

```diff
@@ -0,0 +1,418 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+/* global sinon */
+
+import TestContainer from 'mocha-test-container-support';
+
+import DmnModeler from '../../../src/app/tabs/dmn/modeler/DmnModeler';
+import DrdViewer from '../../../src/app/tabs/dmn/modeler/DrdViewer';
+
+import diagramXML from './diagram.dmn';
+
+const DEFAULT_OPTIONS = {
+  exporter: {
+    name: 'my-tool',
+    version: '120-beta.100'
+  }
+};
+
+const VERY_LOW_PRIORITY = 100;
+
+inlineCSS(require('dmn-js/dist/assets/diagram-js.css'));
+inlineCSS(require('dmn-js/dist/assets/dmn-font/css/dmn-embedded.css'));
+inlineCSS(require('dmn-js/dist/assets/dmn-js-decision-table-controls.css'));
+inlineCSS(require('dmn-js/dist/assets/dmn-js-decision-table.css'));
+inlineCSS(require('dmn-js/dist/assets/dmn-js-drd.css'));
+inlineCSS(require('dmn-js/dist/assets/dmn-js-literal-expression.css'));
+inlineCSS(require('dmn-js/dist/assets/dmn-js-shared.css'));

```

Comment:
Is there any specific reason why we are using CommonJS imports here?

Reply:
It could be changed to

```
import diagramJsCss from 'dmn-js/dist/assets/diagram-js.css';

inlineCSS(diagramJsCss);
```

But that's quite lengthy. In the test setup CSS files are loaded as raw strings, therefore the `inlineCSS` is needed.

Reply:
All right. I guess we're not getting rid of webpack any time soon so it's OK that it stays.